packages/dom/test/unit/lib/ElementAssertion.test.tsx

```diff
@@ -42,4 +53,81 @@ describe("[Unit] ElementAssertion.test.ts", () => {
       });
     });
   });
+
+  describe(".toContainElement", () => {
+    context("when the ancestor element contains the descendant element", () => {
+      it("returns the assertion instance", () => {
+        const { getByTestId } = render(<ContainsTestComponent/>);
+
+        const grandparent = getByTestId("grandparent");
+        const parent = getByTestId("parent");
+        const child = getByTestId("child");
+        const svgElement = getByTestId("svg-element");
+
+        const grandparentTest = new ElementAssertion(grandparent);
+        const parentTest = new ElementAssertion(parent);
+
+        expect(grandparentTest.toContainElement(parent));
+        expect(grandparentTest.toContainElement(child));
+        expect(grandparentTest.toContainElement(svgElement));
+        expect(parentTest.toContainElement(child));
+
+        expect(() => grandparentTest.not.toContainElement(parent))
+          .toThrowError(AssertionError)
+          .toHaveMessage("Expected the container to NOT contain the element");
+
+        expect(() => grandparentTest.not.toContainElement(child))
+          .toThrowError(AssertionError)
+          .toHaveMessage("Expected the container to NOT contain the element");
+
+        expect(() => grandparentTest.not.toContainElement(svgElement))
+          .toThrowError(AssertionError)
+          .toHaveMessage("Expected the container to NOT contain the element");
+
+        expect(() => parentTest.not.toContainElement(child))
+          .toThrowError(AssertionError)
+          .toHaveMessage("Expected the container to NOT contain the element");
+      });
+    });
+
+    context("when the descendant element does not contain the ancestor element", () => {
+      it("returns the assertion instance", () => {
+        const { getByTestId } = render(<ContainsTestComponent/>);
+
+        const grandparent = getByTestId("grandparent");
+        const parent = getByTestId("parent");
+        const child = getByTestId("child");
+        const svgElement = getByTestId("svg-element");
+
+        const parentTest = new ElementAssertion(parent);
+        const childTest = new ElementAssertion(child);
+
+        expect(parentTest.not.toContainElement(grandparent));
+        expect(parentTest.not.toContainElement(svgElement));
+        expect(childTest.not.toContainElement(grandparent));
+        expect(childTest.not.toContainElement(parent));
+        expect(childTest.not.toContainElement(svgElement));
+
+        expect(() => parentTest.toContainElement(grandparent))
```

Comment:
Same here to keep each case (regular/inverted) together. Also create the regular assertion first to maintain consistency with the previous tests.

Reply:
hi @fonsiher, same here. Please, lmk what you think. Thanks again!